prod-aws/otel/otel.tf

```diff
@@ -30,7 +30,13 @@ module "tempo_distributor" {
   consume_topics   = [kafka_topic.otlp_spans.name]
   consume_groups   = ["processor-tempo"]
   cert_common_name = "otel/tempo-distributor"
-  # Limit consuming to 10 MB/s / broker
-  consumer_byte_rate = 10485760
 }
 
+resource "kafka_quota" "tempo_distributor" {
```

Comment:
Same for this one, let's remove it.

Reply:
> Same comment [#609 (comment)](https://github.com/utilitywarehouse/kafka-cluster-config/pull/609#discussion_r1793345653) for this one, let's remove it.

Thanks, I forgot about this one: [4a31df9](https://github.com/utilitywarehouse/kafka-cluster-config/pull/609/commits/4a31df96fbd301798ca9fa909a166975eec8afae)